Code/Editor/EditorFramework/Assets/Implementation/AssetBrowserFilter.cpp

```diff
@@ -255,16 +255,17 @@ bool ezQtAssetBrowserFilter::IsAssetFiltered(ezStringView sDataDirParentRelative
       // if the string is not found in the path, ignore this asset
       if (m_SearchFilter.PassesFilters(sDataDirParentRelativePath) == false)
       {
-        if (pInfo && m_SearchFilter.PassesFilters(pInfo->GetName()) == false)
+        if (pInfo == nullptr)
```

Comment:
how did this bug manifest ?

Reply:
Whenever you started a search all subassets would have been hidden although the name would have matched the search keywords